demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1757,11 +1800,64 @@ export class DemoMeetingApp implements
       return DefaultDeviceController.synthesizeVideoDevice('smpte');
     }
 
-    if (value === 'None') {
-      return null;
+    return value;
+  }
+
+  private videoFilterToProcessor(videoFilter: string): VideoFrameProcessor | null {
+    this.log(`Choosing video filter ${this.videoFilter}`);
+
+    if (videoFilter === 'Emojify') {
+      return new EmojifyVideoFrameProcessor('🚀');
     }
 
-    return value;
+    if (videoFilter === 'CircularCut') {
+      return new CircularCut();
+    }
+
+    if (videoFilter === 'NoOp') {
+      return new NoOpVideoFrameProcessor();
+    }
+
+    return null;
+  }
+
+  private chosenDevice: DefaultVideoTransformDevice;
+  private chosenProcessor: string;
+
+  private async cleanUpChosenTransformDevice(): Promise<void> {
```

Comment:
`stopVideoTransformDevice`?

Reply:
sounds good, also move this call to `leave()`